packages/block-editor/src/components/block-preview/style.scss

```diff
@@ -29,6 +29,17 @@
 	}
 }
 
+.block-editor-block-preview__content {
+	// Resetting the block editor paddings and margins
```

Comment:
I think that's not the ideal fix as I think by default the block editor shouldn't have paddings and margins but It's not easy to reset without big implications. I'm fixing it for the preview right now.